docs/admin/templates/extending-templates/parameters.md

```diff
@@ -204,10 +204,11 @@ operational issue, but it is not advised to overuse this opportunity.
 
 ## Ephemeral parameters
 
-Ephemeral parameters are introduced to users in the form of "build options." Use
-ephemeral parameters to model specific behaviors in a Coder workspace, such as
-reverting to a previous image, restoring from a volume snapshot, or building a
-project without using cache.
+Ephemeral parameters are introduced to users in order to model specific behaviors 
```

Comment:
@EdwardAngert would you mind weighing in on this?

Reply:
Thanks, definitely up for some help here.

The main point @EdwardAngert is that I'd like to remove anything related to build_options. For the rest, really keen for some help but nothing has changed per itself in the logic.